api/openapi_server/controllers/admin_controller.py

```diff
@@ -55,7 +55,7 @@ def initial_sign_in_reset_password():
         'user': user
     }
 
-def removeUser(body: dict):
+def remove_user(body: dict, removeDB: bool = True, removeCognito: bool = True):
```

Comment:
Since all of this method's logic is nested under `if` statements driven by the caller's params, can we log a warning and/or error out if both are passed as false?

Generally speaking, I think it would be nice for some helper methods here for clarity, e.g. `remove_user_from_db(body:dict[str, str])` and `remove_user_from_cognito(body:dict[str, str])` to be called from this method's internals, although this works fine and is still very readable as is currently.